djangocms_versioning/static/djangocms_versioning/js/actions.js

```diff
@@ -19,12 +19,21 @@
                         '">' +
                     '</form>'
                 );
-
+                keepSideFrame = action.attr('class').indexOf('js-versioning-keep-sideframe') === -1
```

Comment:
Add `var`

Reply:
Also, the name suggests value is true when you want to keep the sideframe, but the value is inverted.

Reply:
It should be `!== 1`